locale/en/foundation/case-studies.md

```diff
@@ -2,6 +2,11 @@
 title: Node.js Foundation Resources
 layout: foundation.hbs
 ---
+## Forrester Report: Digital Transformation Using Node.js
+
+Forrester's November 2016 report details five common use cases around Node.js and how it can underpin digital transformation.
+
+* [Read the full report here](https://reprints.forrester.com/#/assets/2/796/'RES135952'/reports)
```

Comment:
@ZibbyKeaton URL should be `https://reprints.forrester.com/#/assets/2/796/RES135952/reports` (without the single quotes), otherwise looks good to me!